backend/src/test/java/com/woowacourse/levellog/domain/TeamTest.java

```diff
@@ -293,8 +294,64 @@ void startAtInvalidDateTime_Exception() {
 
             // when & then
             assertThatThrownBy(() -> team.update(updateTitle, updatePlace, updateStartAt))
-                    .isInstanceOf(InvalidFieldException.class)
+                    .isInstanceOf(InterviewTimeException.class)
                     .hasMessageContaining("잘못된 시작 시간을 입력했습니다.");
         }
     }
+
+    @Nested
+    @DisplayName("closeInterview 메소드는")
+    class CloseInterview {
+
+        @Test
+        @DisplayName("팀 인터뷰를 종료 상태로 바꾼다.")
+        void closeInterview_success() {
+            // given
+            final String title = "네오와 함께하는 레벨 인터뷰";
+            final String place = "선릉 트랙룸";
+            final LocalDateTime startAt = LocalDateTime.now().plusDays(3);
+            final String profileUrl = "profile.img";
+            final Team team = new Team(title, place, startAt, profileUrl);
```

Comment:
```suggestion
            final LocalDateTime startAt = LocalDateTime.now().plusDays(3);
            final Team team = new Team("네오와 함께하는 레벨 인터뷰", "선릉 트랙룸", startAt, "profile.img");
```
여기서는 title, place, profileUrl 보다는 startAt이 중요한 것 같아서 해당 부분만 변수로 추출해놓는 것은 어떨까요?😊